docs/topics/rn-new-features.adoc

```diff
@@ -8,35 +8,31 @@
-.Rules for migrating from Java EE8 to Jakarta EE9
-{ProductShortName} rules replace Java EE8 artifact dependencies and package import statements and rename XML schema namespaces, prefixed properties, and bootstrapping files for Jakarta EE9. You run these rules by specifying the `jakarta-ee` target.
+.Analyze Known Libraries
+You can now use the `analyzeKnownLibraries` advanced option to analyze libraries embedded within your applications. By default, MTA's default behavior is to only analyze application code, rather than libraries that are declared as dependencies. This option extends your ability to analyze your applications.
 
-.OpenRewrite recipe support
+.Rules for migrating from OpenJDK 8 to OpenJDK 11
+MTA now includes rules to help you migrate applications from OpenJDK 8 (Java 8) to 11.
 
-link:https://docs.openrewrite.org/[OpenRewrite] automates large-scale, distributed source code refactoring. You can run OpenRewrite recipes by using the link:{ProductDocUserGuideURL}[{ProductShortName} CLI].
+.Technology Reports
+The technology reports have been enriched. New technology classifications have been introduced, many new technologies are now identified and new technology tags have been generated.
 
-The recipe `org.jboss.windup.JavaxToJakarta` renames imported `javax` packages to their `jakarta` equivalents.
+.Apply all Quick Fixes
+In the the MTA IDE plugins and extensions, you can now apply all suggested Quick Fixes at once, rather than one-by-one.
 
-.MTA extension for Visual Studio Code compatible with Codespaces
+.Rules for migrating from Spring Boot to Quarkus
+Additional rules have been added to support migrating from Spring Boot to Quarkus.
 
-The {ProductShortName} extension for Microsoft Visual Studio Code is now compatible with Microsoft Visual Studio Codespaces.
+.Rules for migrating from IBM WebSphere to Open Liberty
+A comprehensive collection of rules has been added to support migrating traditional IBM WebSphere applications to Open Liberty.
 
-.Rules for the Springboot-to-Quarkus migration path
+.New OpenRewrite capabilities including additional recipes
+* The integration of MTA with OpenRewrite has been extended to support multiple migration paths.
+* An additional recipe for the the `javax` to `jakarta` transformation path has been added, as well as the first Spring Boot to Quarkus recipe.
 
-Rulesets identify Quarkus-supported technologies by detecting a POM dependency, embedded JAR file, or a partial file path within an Apache Maven Shade uber JAR.
-
-The rules identify the following technologies:
+[IMPORTANT]
+====
+OpenRewrite is provided as Technology Preview only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs), might not be functionally complete, and Red Hat does not recommend to use them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
 
```

Comment:
@RichardHoch 
The last sentence is a bit awkward - twice 'features' and twice 'provide'. I would suggest:
"Technology Preview allows early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process."

Reply:
@sbeskin-redhat This is a set paragraph. We might want to change it later, but not now.